### PR TITLE
Depends on logstash-core >= 2.0 and update to a major version

### PR DESCRIPTION
**Pull request by ph**

### CODE REVIEW

File: logstash-mixin-aws.gemspec [hunk 1]
```diff
@@ -16,7 +16,7 @@ Gem::Specification.new do |s|
   s.test_files = s.files.grep(%r{^(test|spec|features)/})
 
   # Gem dependencies
-  s.add_runtime_dependency "logstash-core", '>= 1.4.0', '< 2.0.0'
+  s.add_runtime_dependency "logstash-core", ">= 2.0.0", "< 3.0.0"
```
Comment (suyograo) [inline logstash-mixin-aws.gemspec]: why the change to double quotes?

Comment (ph) [inline logstash-mixin-aws.gemspec]: Well, I see few reason for this.

1. There is no real benefits of using single quote..
2. We have a mix and match of single quote / double quote, I though we had decided to use double quotes ;)
3. And it was a bit easier on my side for the regular expression

  content.gsub!(/add_runtime_dependency ("|')logstash-core("|').+/, 'add_runtime_dependency "logstash-core", ">= 2.0.0", "< 3.0.0"')

Comment (suyograo) [inline logstash-mixin-aws.gemspec]: cool, just wanted to raise this

Comment (ph) [inline logstash-mixin-aws.gemspec]: So its a LGTM to run on **all** plugins?

**Comment by suyograo** [comment]
LGTM

File: logstash-mixin-aws.gemspec (hunk 1 above)
Comment (jordansissel) [inline logstash-mixin-aws.gemspec]: @ph LGTM

**Comment by ph** [comment]
Done with the mass update script.
